src/freenas/usr/local/lib/middlewared_truenas/plugins/enclosure_/map.py

```diff
@@ -0,0 +1,108 @@
+from collections import namedtuple
+import re
+
+from middlewared.service import Service, private
+
+ProductMapping = namedtuple("ProductMapping", ["product_re", "mappings"])
+VersionMapping = namedtuple("VersionMapping", ["version_re", "slots"])
+MappingSlot = namedtuple("MappingSlot", ["num", "slot", "identify"])
+
+
+MAPPINGS = [
+    ProductMapping(re.compile(r"To Be Filled By O.E.M."), [
```

Comment:
Why this?

Reply:
That's what was needed to test on test06, do you want me to remove that?

Reply:
Yes, I was thinking just was just be there while testing in the box, not actually commit it to the repo. Or wait until we have a final say of what the system-productname will be